docs/en/ingest-management/getting-started.asciidoc

```diff
@@ -192,6 +192,78 @@ to the data stream.
 
 //TODO: Add dashboard.
 
+[discrete]
+== Data from Splunk (Experimental)
+
+Apache, AWS Cloudtrail, Nginx, and Zeek integrations offer the ability
+to pull logs from a Splunk Enterprise instance.  This sections covers
+configuration & limitations of this option.
```

Comment:
Slight adjustment: 'Apache, AWS Cloudtrail, Nginx and Zeek integrations offer the ability to seamlessly ingest data from a Splunk Enterprise instance, making it available for rapid analysis in Elastic.

Maybe just include a sentence on how it works at a high-level. e.g. The integration works via an input within Elastic packages, which connects to the Splunk REST API to pull the relevant data, based on a search string.

Reply:
also added image showing components